src/components/form-helpers/captcha-utils.tsx

```diff
@@ -6,7 +6,7 @@ type CaptchaUtils = Readonly<
   [
     (props: any) => JSX.Element,
     () => Promise<CaptchaVerificationResult>,
-    React.RefObject<ReCAPTCHA>,
+    React.RefObject<ReCAPTCHA>
```

Comment:
Please revert this change, it seems that it is being generated by a wrong _Prettier_ version.

Try this:

```bash
$ rm -rf  node_modules
$ yarn
$ yarn prettier
```